### ISSUE TITLE
Intrusion Detection pagination is deffective

### ISSUE THREAD

**Issue by oparoz** [opnsense/core#756]
### Steps to reproduce
1. Go to the Alerts tab
1. Let's say you have 10 entries. By default you'll be shown 7 and there will be 2 pages of logs
1. Change the view to show 50 entries and press the search button. (Would be nice if a button press wasn't required)

### Expected result

* 10 entries on the page
* 1 page only in the pagination, bottom left
* *Showing 1 to 10 of 10 entries*, bottom right

### Actual result

* 10 entries on the page
* **2 pages in the pagination, bottom left**
* ***Showing 1 to 7 of 10 entries*, bottom right**

**Comment by AdSchellevis** [comment]
Due to the often large file, the pagination is always a guestimate of the actual results. The backend process parses the data until it found requested number and exits his search, without knowing about the possible rest of them (this would just take too much time).
The bootgrid component we use unfortunately doesn't seem to handle the response for the last page very well if the number of results equals the total number.

At first we implemented the search without the button, but you really don't want that... it can get very busy in different processes doing a search on a very large file, so we choose to explicitly ask for a refresh on all changes here.

Long story short, the pagination is probably never going to be perfect for this type of operation.

**Comment by AdSchellevis** [comment]
timeout
